src/app/portlets/dot-edit-page/content/components/dot-edit-page-workflows-actions/dot-edit-page-workflows-actions.spec.ts

```diff
@@ -114,6 +125,7 @@ describe('DotEditPageWorkflowsActionsComponent', () => {
         beforeEach(() => {
             // spyOn(dotWorkflowService, 'getContentWorkflowActions').and.returnValue(Observable.of([]));
             spyOn(dotWorkflowService, 'getContentWorkflowActions').and.callThrough();
+            component.ngOnChanges();
```

Comment:
In the past, we run into several issue by calling ngOnChanges manually, angular provide a way to test this component inside a host component, here is some doc: https://angular.io/guide/testing#component-inside-a-test-host

And here is a way we did it: https://github.com/dotCMS/core-web/blob/master/src/app/portlets/dot-edit-page/layout/components/dot-edit-layout-grid/dot-edit-layout-grid.component.spec.ts#L27

Let me know if you run into issues and I can walk you through.

Reply:
done